examples/completions.c

```diff
@@ -9,34 +9,32 @@
 #include <linux/printk.h>
 #include <linux/version.h>
 
-static struct {
-    struct completion crank_comp;
-    struct completion flywheel_comp;
-} machine;
```

Comment:
You should also modify the passage in the Completions section:
> The machine structure stores the completion states for the two threads. At the exit point of each thread the respective completion state is updated, and wait_for_completion is used by the flywheel thread to ensure that it does not begin prematurely.

Reply:
Updated tex file to match code
-- Explained parts of a completions (init, wait_for_completion & complete)
-- Clarified the crank and flywheel threads better in the text